--- glibc-2.27.orig/debian/rules
+++ glibc-2.27/debian/rules
@@ -140 +140 @@
-  DEB_INDEP_REGULAR_PACKAGES = locales
+  DEB_INDEP_REGULAR_PACKAGES = 
@@ -144 +144 @@
-    DEB_ARCH_REGULAR_PACKAGES += locales-all
+    DEB_ARCH_REGULAR_PACKAGES += 
--- glibc-2.27.orig/localedata/Makefile
+++ glibc-2.27/localedata/Makefile
@@ -196 +196 @@
-include SUPPORTED
+SUPPORTED-LOCALES = C.UTF-8/UTF-8
--- glibc-2.27.orig/debian/control.in/libc
+++ glibc-2.27/debian/control.in/libc
@@ -7,7 +7,7 @@
 Suggests: glibc-doc, debconf | debconf-2.0, locales [!hurd-i386]
 Provides: libc6-sparcv9b [sparc sparc64], libc0.1-i686 [kfreebsd-i386], libc0.3-i686 [hurd-i386], libc6-i686 [i386], libc6-xen [i386], libc6-armel [armel], libc6-armhf [armhf]
 Conflicts: libc6-loongson2f [mipsel], libc0.1-i686 [kfreebsd-i386], libc6-i686 [i386], openrc (<< 0.27-2~), libc6-xen [i386]
-Breaks: locales (<< @GLIBC_VERSION@), locales-all (<< @GLIBC_VERSION@), hurd (<< 1:0.5.git20140203-1), libtirpc1 (<< 0.2.3)
+Breaks: hurd (<< 1:0.5.git20140203-1), libtirpc1 (<< 0.2.3)
 Replaces: libc6-amd64 [amd64],
           libc6-i386 [i386],
           libc0.1-i686 [kfreebsd-i386],
