--- glibc-2.27.orig/debian/control.in/main
+++ glibc-2.27/debian/control.in/main
@@ -1,16 +1,16 @@
 Source: glibc
 Section: libs
 Priority: required
-Build-Depends: gettext, dpkg (>= 1.18.7), dpkg-dev (>= 1.17.14), xz-utils, file, quilt,
- autoconf, gawk, debhelper (>= 10), rdfind, symlinks, netbase, gperf, bison,
+Build-Depends: gettext, dpkg, dpkg-dev (>= 1.17.14), xz-utils, file,
+ autoconf, gawk, debhelper, rdfind, symlinks, netbase, bison,
  linux-libc-dev (>= 3.9) [linux-any], systemtap-sdt-dev [linux-any],
  libaudit-dev [linux-any], libcap-dev [linux-any], libselinux-dev [linux-any],
  mig (>= 1.5-3) [hurd-i386], gnumach-dev (>= 2:1.8+git20170102-1~) [hurd-i386],
  hurd-dev (>= 1:0.9.git20180305~) [hurd-i386] | hurd-headers-dev [hurd-i386],
  hurd-dev (>= 1:0.9.git20180305~) [hurd-i386] <!stage1> | libihash-dev [hurd-i386] <!stage1>,
  kfreebsd-kernel-headers [kfreebsd-any],
- binutils (>= 2.25), binutils (>= 2.29) [amd64 i386 x32],
- g++-7, g++-7 (>= 7.2.0-20) [amd64 i386 x32], g++-7-multilib [amd64 armhf armel i386 kfreebsd-amd64 mips mipsel mipsn32 mipsn32el mips64 mips64el mipsr6 mipsr6el mipsn32r6 mipsn32r6el mips64r6 mips64r6el powerpc ppc64 s390x sparc sparc64 x32] <!nobiarch>,
+ binutils (>= 2.25),
+ g++-7-multilib [amd64 armhf armel i386 kfreebsd-amd64 mips mipsel mipsn32 mipsn32el mips64 mips64el mipsr6 mipsr6el mipsn32r6 mipsn32r6el mips64r6 mips64r6el powerpc ppc64 s390x sparc sparc64 x32] <!nobiarch>,
  python3:native <!nocheck>,
  libc-bin (>= @GLIBC_VERSION@) <cross>
 Build-Depends-Indep: perl, po-debconf (>= 1.0)
